android/build.gradle

```diff
@@ -120,6 +120,7 @@ dependencies {
     // Supporting libraries.
     implementation("io.coil-kt:coil-compose:2.6.0")
     implementation("com.google.zxing:core:3.5.1")
+    implementation("com.patrykandpatrick.vico:compose:2.0.0-alpha.21")
```

Comment:
There is a stable 1.15 version of this.  Is that suitable over the alpha?

There is also Apache licenses so we need to include the proper attributions.

Reply:
Downgraded. https://github.com/tailscale/corp/issues/21114 tracks updating the LICENSES